space-dodge-game/README.md

```diff
@@ -21,62 +22,52 @@ CC BY-SA 3.0
 
 There are 5 versions of the code.
 
-The master version includes the base code before any optimisations are
+The base version includes the base code before any optimisations are
 applied.
 
-The four Crosswalk* versions have different code, depending
-on the version of Crosswalk used and the approach used to
-fit the application into the device screen:
+The other four versions have different code, depending on
+the approach used to fit the application into the device screen:
 
-*   Crosswalk-6-scale
+*   screen-orientation-scale
 
-    *   landscape orientation set with `screen.lockOrientation()`
-    *   fullscreen set with `--fullscreen` option to make_apk.py
+    *   landscape orientation set with `screen.orientation.lock()`
+    *   fullscreen set with `document.documentElement.webkitRequestFullScreen()`
     *   whole game scaled in CSS to fit screen
 
-*   Crosswalk-6-resize
+*   screen-orientation-resize
 
-    *   landscape orientation set with `screen.lockOrientation()`
-    *   fullscreen set with `--fullscreen` option to make_apk.py
+    *   landscape orientation set with `screen.orientation.lock()`
+    *   fullscreen set with `document.documentElement.webkitRequestFullScreen()`
     *   game elements resized to fit screen
 
-*   Crosswalk-8-scale
+*   manifest-orientation-scale
 
     *   landscape orientation and fullscreen set in manifest
     *   `xwalk_launch_screen` enabled in manifest
     *   whole game scaled in CSS to fit screen
 
-*   Crosswalk-8-resize
+*   manifest-orientation-resize
 
     *   landscape orientation and fullscreen set in manifest
     *   `xwalk_launch_screen` enabled in manifest
     *   game elements resized to fit screen
 
-The Crosswalk-6* versions should also work with Crosswalk 5 and 7 (6
-is the version they were tested with); the Crosswalk-8* versions should
-work with Crosswalk version 8 or above.
+All the 5 versions should work with latest Crosswalk Project.
 
 ## Android packages
 
 To create an Android package for the game, follow the instructions at:
 https://crosswalk-project.org/#documentation/getting_started/run_on_android
 
-You will need the correct version of Crosswalk for the version of
-the code you intend to package (see above).
+    python make_apk.py --package=org.crosswalkproject.spacedodge \
+      --manifest=screen-orientation-scale/manifest.json
 
-When building the Crosswalk 6 versions, you will need to pass the
-`--fullscreen` option for the game to run in fullscreen, e.g.
+    python make_apk.py --package=org.crosswalkproject.spacedodge \
```

Comment:
I think this fullscreen work for app framework, not html document element, we require a document's top-level browsing context to be fullscreen. This means we must change to fullscreen by manual becasue FullScreen API needs user interaction.
For manifest ones, we can add the config in manifest.json:
```
"display" : ["fullscreen"],
"orientation" : "landscape"
```